### PR TITLE
PLAT-129803: Fix LabeledIconButton shape to circular

### PR DESCRIPTION
**Pull request by MikyungKim**
This PR fixes LabledIconButton shape to circular.
And the warning saying 
```
Warning: React does not recognize the `iconOnly` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `icononly` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```
Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### CODE REVIEW

File: LabeledIconButton/LabeledIconButton.js [hunk 1]
```diff
@@ -130,6 +140,7 @@ const LabeledIconButtonBase = kind({
 					backgroundOpacity={backgroundOpacity}
 					icon={icon}
 					iconComponent={iconComponent}
+					iconOnly
```
Comment (jeonghee27) [inline LabeledIconButton/LabeledIconButton.js:143]: I thinks issue can be fixed by only this line.

File: LabeledIconButton/LabeledIconButton.js [hunk 2]
```diff
@@ -83,6 +83,14 @@ const LabeledIconButtonBase = kind({
 		 */
 		iconComponent: EnactPropTypes.component,
 
+		/**
+		 * True if button is an icon only button.
+		 *
+		 * @type {Boolean}
+		 * @private
+		 */
+		iconOnly: PropTypes.bool,
```
Comment (jeonghee27) [inline LabeledIconButton/LabeledIconButton.js:92]: I don't know why we should add prop here. Do we have to add this for samplers?

File: LabeledIconButton/LabeledIconButton.js (hunk 1 above)
Comment (MikyungKim) [reply LabeledIconButton/LabeledIconButton.js:143]: The issue could be fixed by this line but I'd like to fix the warning when the story loaded.

File: LabeledIconButton/LabeledIconButton.js (hunk 2 above)
Comment (MikyungKim) [reply LabeledIconButton/LabeledIconButton.js:92]: As I was saying, to fix the warning, I need `delete rest.iconOnly;` line and it needs this propTypes for lint warning.

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**